docs/modeling/backing-field.rst

```diff
@@ -0,0 +1,45 @@
+.. include:: /_shared/rc1-notice.txt
+
+Backing Fields
+==============
+
+When a backing field is configured, EF will write directly to that field when materializing entity instances from the database (rather than using the property setter). This is useful when there is no property setter, or the setter contains logic that should not be executed when setting initial property values for existing entities being loaded from the database.
+
+.. caution::
+  The ``ChangeTracker`` has not yet been enabled to use backing fields when it needs to set the value of a property. This is only an issue for foreign key properties and generated properties - as the change tracker needs to propagate values into these properties. For these properties, a property setter must still be exposed.
+
+  `Issue #4461 <https://github.com/aspnet/EntityFramework/issues/4461>`_ is tracking enabling the ``ChangeTracker`` to write to backing fields for properties with no setter.
+
+.. contents:: In this article:
+  :depth: 2
+  :local:
+
+Conventions
+-----------
+
+By convention, the following fields will be discovered as backing fields for a given property (listed in precedence order):
+  * <propertyName> differing only by case
+  * _<propertyName>
+  * m_<propertyName>
```

Comment:
This is slightly simplified from the actual rules, as we have some additional precedence around casing of the field name... but it seems like such a corner case to have two fields that match a property name and only differ by case that I chose to keep it simple.